content/company-info-and-process/policies/access-control-policy.md

```diff
@@ -58,7 +58,7 @@ Control and management of individual user passwords is the responsibility of all
 
 ## Credentials Management Policy
 
-Keep in mind that the passwords we choose contribute to the security of our Sourcegraph-managed systems and data. In addition to being users, many of us are also Administrators and so it is especially important to be thoughtful when selecting a password. \
+The passwords we choose contribute to the security of our Sourcegraph-managed systems and data. The use of poor credentials in service and system can lead to disclosure of sensitive information and data breaches. In addition to being users, many of us are also Administrators and so it is especially important to be thoughtful when selecting a password. \
```

Comment:
Moved "The use of poor credentials..." to here from the opening paragraph because it fits better here and keeps the opening paragraph focused on the purpose of the policy.